website/src/app/Components/HomeComponent.tsx

```diff
@@ -7,17 +7,33 @@ interface TokenData {
   newCount: number
 }
 
+interface CommandOption {
+  name: string
+  requiresArgument: boolean
+}
+
+type CommandOptions = {
+  [key: string]: CommandOption
+}
+
 const HomeComponent: React.FC = () => {
   const [serialNumber, setSerialNumber] = useState<string>("")
   const [counter, setCounter] = useState<number | null>(null)
   const [startingCode, setStartingCode] = useState<number | null>(null)
   const [privateKey, setPrivateKey] = useState<string>("")
-  const [commandArgument, setCommandArgument] = useState<number | null>(null)
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [selectedCommand, setSelectedCommand] = useState<string>("1")
+  const [commandArgument, setCommandArgument] = useState<number>(7)
   const [timeGranularity, setTimeGranularity] = useState<number | null>(null)
   const [result, setResult] = useState<TokenData | null>(null)
   const [error, setError] = useState<string | null>(null)
 
+  const commandOptions: CommandOptions = {
+    "1": { name: "add_time - Add PAYG time", requiresArgument: true },
+    "2": { name: "set_time - Set PAYG time", requiresArgument: true },
+    "3": { name: "disable_payg - Disable PAYG", requiresArgument: false },
+    "4": { name: "counter_sync - Counter sync", requiresArgument: false },
```

Comment:
I _think_ `counter_sync` does require an argument. I.e. the counter to sync with. Is there a way for you to test this?

Reply:
@dmohns fixed